### ISSUE TITLE
Feedback seems broken

### ISSUE THREAD

**Issue by joeyklee** [moovel/lab-what-the-street#124]
It seems for a lot of cities the result is always the same:

# Portland
<img width="1552" alt="screen shot 2017-09-19 at 16 59 54" src="https://user-images.githubusercontent.com/3622055/30599279-4659ed4e-9d5c-11e7-9509-42ddc7c0b5f8.png">

# Stuttgart 

<img width="1552" alt="screen shot 2017-09-19 at 17 00 05" src="https://user-images.githubusercontent.com/3622055/30599288-4c57d3fa-9d5c-11e7-9d05-24be336069ec.png">

# New York

<img width="1552" alt="screen shot 2017-09-19 at 17 00 16" src="https://user-images.githubusercontent.com/3622055/30599494-f0ae42fe-9d5c-11e7-8582-519d5d3fdd1a.png">

BUT for Jakarta, we still get useful feedback... 

<img width="1552" alt="screen shot 2017-09-19 at 17 00 25" src="https://user-images.githubusercontent.com/3622055/30599543-20113312-9d5d-11e7-8011-59cd1fe588eb.png">

Could it be that the scripts for calculating the feedback are broken somehow?

-J

**Comment by mszell** [comment]
I don't think it is broken. The result fetches something like the last 1000 guesses. Once a city has reached that number, the distribution shouldn't change much afterwards. Jakarta just hasn't reached it yet.

**Comment by joeyklee** [comment]
@mszell - yeah, but tbh, it is really confusing because even if you guess almost perfectly, the result tells you the same as if you guess really poorly.

**Comment by mszell** [comment]
Ohh, you mean that feedback. Ok, probably one could finetune it somehow.

**Comment by joeyklee** [comment]
@mszell - is it to finetune it or rather fix it? The result is the same whether you guess terribly or if you guess well. 

Also, how can you be a "mobility guru" but then get "unfortunately your guess was far off"...? 

For stuttgart:

<img width="1380" alt="screen shot 2017-09-26 at 08 58 38" src="https://user-images.githubusercontent.com/3622055/30846802-422825a0-a299-11e7-8716-a518be59e925.png">

vs. 

<img width="1388" alt="screen shot 2017-09-26 at 08 59 18" src="https://user-images.githubusercontent.com/3622055/30846809-4797fe84-a299-11e7-9299-5570988c5f10.png">

**Comment by mszell** [comment]
Looks like a bug - it's hard to do better than that :)
